### ISSUE TITLE
Method 'OnAttachBinaryFile' obsolete

### ISSUE THREAD

**Issue by apoyas** [microsoft/ALAppExtensions#8692]
Hello!

BC16
There is an integration event 'OnAttachBinaryFile' in table 133 "Incoming Document Attachment".
Unfortunately it is marked as obsolete.
```AL
[IntegrationEvent(true, false)]
[Obsolete('Function scope will be changed to OnPrem','15.1')]
procedure OnAttachBinaryFile()
begin
end;
```

What should be used as a replacement method for the function and when it is scheduled to be available?

Thanks!

**Comment by StanPesotskiy** [comment]
New event OnAfterImportAttachment of codeunit 134 "Import Attachment - Inc. Doc." could be used instead of obsoleted one. It will be available from update 17.1

**Comment by bc-ghost** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. We will update this issue with information about availability.
